queries/futures/types.ts

```diff
@@ -25,37 +25,6 @@ export type MarginTransfer = {
 	asset?: FuturesMarketAsset;
 };
 
-export type PositionHistory = {
-	id: Number;
-	transactionHash: string;
-	timestamp: number;
-	openTimestamp: number;
-	closeTimestamp: number | undefined;
-	market: string;
-	asset: FuturesMarketAsset;
-	account: string;
-	abstractAccount: string;
-	accountType: FuturesAccountType;
-	isOpen: boolean;
-	isLiquidated: boolean;
-	size: Wei;
-	feesPaid: Wei;
-	netFunding: Wei;
-	netTransfers: Wei;
-	totalDeposits: Wei;
-	initialMargin: Wei;
-	margin: Wei;
-	entryPrice: Wei;
-	avgEntryPrice: Wei;
-	exitPrice: Wei;
-	leverage: Wei;
-	side: PositionSide;
-	pnl: Wei;
-	pnlWithFeesPaid: Wei;
-	totalVolume: Wei;
-	trades: number;
-};
-
 export enum PositionSide {
 	LONG = 'long',
 	SHORT = 'short',
```

Comment:
Looked like we were still importing this one in some places, might be worth removing this as well and always importing from SDK.